crates/chia-protocol/src/bytes.rs

```diff
@@ -19,10 +19,19 @@ use pyo3::prelude::*;
 #[cfg(feature = "py-bindings")]
 use pyo3::types::PyBytes;
 
+#[cfg(not(feature = "serde"))]
 #[derive(Default, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
 #[cfg_attr(feature = "arbitrary", derive(arbitrary::Arbitrary))]
 pub struct Bytes(Vec<u8>);
 
+#[cfg(feature = "serde")]
```

Comment:
I'm not a fan of duplicating the definition of `Bytes` and `BytesImpl`. It might be better to get rid of `serde_with` entirely and just use the manual implementation like I did for `PublicKey`